File: WorkDB/WSClientDB/dataClasses/DataOfDialog.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace WSClientDB.dataClasses
{
    class DataOfDialog
    {
        public string dialog_id { get; set; }
        public string tagUser { get; set; }
        public int enteredTime { get; set; }
        public int countMsg { get; set; }
        public int lastTimeMsg { get; set; }
        public int typeOfDlg { get; set; }
        public int rang { get; set; }
        public string nameOfChat { get; set; }
    }
}

File: Client/app/src/main/java/com/example/myapplication/dataClasses/DataOfDialog.kt
package com.example.myapplication.dataClasses

import kotlinx.serialization.Serializable

@Serializable
data class DataOfDialog(
    val dialog_id: String,
    val tagUser: String,
    val enteredTime: Int,
    val countMsg: Int,
    val lastTimeMsg : Int,
    val typeOfDlg: Int,
    val rang: Int,
    val nameOfChat: String
)
